ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/model/AdditionalData.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
check for other places as well.
```suggestion
 *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Actually this code is written in last year, thats why it has `2023`. Thats fine right?

Reply:
But, we are merging this to master in this year. IMO in that case we can argue that the licensing header should be updated too. But, this is a personal preference not a standard.

Reply:
Fixed via [f8e7bf9](https://github.com/ballerina-platform/openapi-tools/pull/1564/commits/f8e7bf96e0493f6eca41f26add8e30b9e6c3cd93)